languages/csharp/csharp.go

```diff
@@ -19,11 +31,30 @@ func (language) Aliases() []string {
 }
 
 func (l language) GetLanguage() *sitter.Language {
-	return java.GetLanguage()
+	return csharp.GetLanguage()
 }
 
 func (l language) Imports(content []byte, n *sitter.Node) ([]string, error) {
-	log.Println(n)
+	var err error
```

Comment:
Please make it a global as well (`qerr`?).
In other cases it will print an error the first time, and then fail with NPE on the second try (`once` is not called, `err` is nil).

Reply:
right, done.